bin/critterget.py

```diff
@@ -801,13 +818,17 @@ def getCredentials(sessionKey):
 
         # return first set of credentials
         if (debug) : print "Entities is ", entities
-        if entities is not None:
+        if entities:
             for i, c in entities.items():
```

Comment:
Why are we referencing i here?  Also, what's up with the letters i and c?
Why are we redefining auth multiple times?
Do we not know what the key name is for the parent?
Assuming we want the first item, and we don't know the key name, something like this seems reasonable, but not sure of the use case:
if entities:
  auth = entities.items()[0][1].get('clear_password')

If we know the key we want, it should be much easier though.